go.mod

```diff
@@ -4,8 +4,10 @@ go 1.16
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0
+	github.com/IABTechLab/adscert v0.34.0
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
+	github.com/benbjohnson/clock v1.3.0
```

Comment:
Nice. i think I'll use this in the pubstack tests too.

Reply:
I stole it from a draft branch you shared with me :)